website/src/mdRoutes.js

```diff
@@ -62,6 +62,15 @@ import responsiveVis from '../../docs/examples/responsive-vis.md';
 import zoomableChart from '../../docs/examples/zoomable-chart.md';
 import gitHistory from '../../docs/examples/history-example.md';
 
+// Ocular searches for links with at least one '/' and replaces them
+// with the first route it finds that includes the same markdown name
+// (that is error prone, and that's why current Axes links will all
+// point to Example Axes page instead of Documentation Axes page)
+const fixMarkdownLinks = (markdown) => {
+  const markdownLinksRegex = /(?:\(([^()/\n]+\.md)\))/g;
+  return markdown.replace(markdownLinksRegex, '(/$1)');
+};
+
 export default [
```

Comment:
could you apply fixMarkdownLinks as a map across each of the pages and children? DRYer that way

Reply:
Sure! Thanks

Reply:
Do you mean to apply a .map on each children array? or iterating over all the structure? :thinking:

Reply:
I was imagining something like iterating over the whole structure, eg at the end add a .map(section => section.children.map(..))

Reply:
:+1: Ok